src/components/CommentPageButtons.jsx

```diff
@@ -0,0 +1,23 @@
+import { fetchComments } from "../utils/apicalls"
+
+export const CommentPageButtons = ({totalComments, setComments, article_id}) => {
+    const handleClick = (event) => {
+        const page = event.target.id
+        fetchComments(article_id, page).then((body) => {
+            setComments(body.data.comments)
+        })
+    }
+
+    const buttonArray = []
```

Comment:
This code feels a little clunky, although I can see the intention. 
What about the articles in the database that have fewer than 10 comments - your labelling shows 
"showing 10 of 7 comments" - and it creates a slightly hacky feel to the comments list. 

Also, consider using the length of the array of comments that you get back from the api to provide the value for the number of comments, you're having to pass the totalComments value through several layers of your component tree but you can derive the information from the comments themselves.